srcpkgs/apparmor/template

```diff
@@ -18,7 +18,7 @@ license="GPL-2.0-only"
 homepage="https://gitlab.com/apparmor/apparmor"
 changelog="https://gitlab.com/apparmor/apparmor/-/wikis/Release_Notes_${version}"
 distfiles="https://gitlab.com/apparmor/apparmor/-/archive/v${version}/apparmor-v${version}.tar.gz"
```

Comment:
if we use `https://launchpad.net/apparmor/3.1/3.1.2/+download/apparmor-3.1.2.tar.gz`, then we can drop autoreconf because this archive contains a pre-generated configure file. Then automake and libtool in hostmakedepends can probably be removed.

Reply:
I already tried that, but I was getting configure errors with aarch64 cross-compile (something to do with Perl). I'll take a closer look later.

Reply:
You can just add perl to hostmakedepends